src/functTest/groovy/com/bmuschko/gradle/docker/DockerNettyDockerCmdExecFactoryFunctionalTest.groovy

```diff
@@ -0,0 +1,57 @@
+package com.bmuschko.gradle.docker
+
+import com.github.dockerjava.api.DockerClient
+import com.github.dockerjava.netty.NettyDockerCmdExecFactory
+import org.gradle.testkit.runner.BuildResult
+import spock.lang.Ignore
+
+class DockerNettyDockerCmdExecFactoryFunctionalTest extends AbstractGroovyDslFunctionalTest {
```

Comment:
You will want to add a test case for the negative case. If none of your Netty properties is set, then it should use the Jersey implementation and not render the message.

Reply:
Done.